engine/src/flutter/shell/platform/windows/flutter_windows_unittests.cc

```diff
@@ -131,20 +131,23 @@ TEST_F(WindowsTest, LaunchHeadlessEngine) {
   ASSERT_NE(engine, nullptr);
 
   std::string view_ids;
-  fml::AutoResetWaitableEvent latch;
+  bool signalled = false;
```

Comment:
Just one "l" for US English :)

```suggestion
  bool signaled = false;
```